Front top tag

naoi217 opened the pull request:
## やったこと
topページのフロント対応をした。
- タグの作成
- ボイスカードの作成
- 共通ヘッダーの作成

レビュー観点
UI図：https://drive.google.com/file/d/1NWRm-PIzBgNPBOBcwp4WazugJFkZsbb9/view?usp=sharing
を元に表示崩れや不備がないかチェック

HTML、CSSの書き方がパッと見きもくないか。

naoi217 committed:
add tag

naoi217 committed:
add top &vue-router

naoi217 committed:
add reset css

shamisonn commented on src/components/Tag.vue:
```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="tag">
+    <a href="#" class="btn-icon"><p class="tag-name"># {{ tagName }}</p></a>
```
下記でもいい気がしたけど何か理由ってあったりする？
```suggestion
    <a href="#" class="btn-icon"><p class="tag-name">#{{ tagName }}</p></a>
```

shamisonn commented on src/components/Tag.vue:
```diff
@@ -0,0 +1,42 @@
+  margin: 2px 4px;
+}
+
+.tag-name {
+  font-size: 0.5em;
+  padding: 0.5em;
+}
```
ちょっと上下の空白がきになったりしたので、こんな感じにしても良さそうにみえました！
```suggestion
.tag-name {
  font-size: 0.5em;
  padding: 0.5em;
  margin: 0;
}
```

shamisonn commented on src/components/Top.vue:
```diff
@@ -0,0 +1,28 @@
+<template>
+  <div id="top">
+    <!-- TODO: v-forにする。href動的に生成する。 -->
```
👍

shamisonn reviewed (changes requested):
> HTML、CSSの書き方がパッと見きもくないか。

scoped cssを使っていてめっちゃ見やすかったです！Vueすき！！！いいっすね！

> UI図：https://drive.google.com/file/d/1NWRm-PIzBgNPBOBcwp4WazugJFkZsbb9/view?usp=sharing
を元に表示崩れや不備がないかチェック

こっちの点でちょっとだけ細かいところ書いてみました！
提案程度に見てもらって、もとのほうがいいなーってなったらその旨コメントください！

naoi217 committed:
指摘の修正

https://github.com/sakaba-project/sakaba-front/pull/16#discussion_r449882827

https://github.com/sakaba-project/sakaba-front/pull/16#discussion_r449883038

naoi217 commented:
再レビューをお願いしまああす

shamisonn commented:
@naoi217 LGTM!
